pkg/users/client.go

```diff
@@ -238,6 +238,44 @@ type ListAuthFactorsResponse struct {
 	ListMetadata common.ListMetadata `json:"listMetadata"`
 }
 
+type InviteObject struct {
+	Object         string `json:"object"`
```

Comment:
```suggestion
type Invite struct {
```
small naming change + looks like we omit the object field for other object types.